packages/rn-tester/js/examples/Accessibility/AccessibilityExample.js

```diff
@@ -856,15 +907,15 @@ class EnabledExample extends React.Component<{}> {
     this._subscription?.remove();
   }
 
-  _handleToggled = isEnabled => {
+  _handleToggled = () => {
```

Comment:
Changing this to not take a parameter changes the meaning. What was the reason you had for removing the parameter?

This might be out of scope for this PR as I'm honestly not sure what this handler is doing. It seems to be used for both listening to changes to the desired accessibility feature, and also handling the onPress of the button.

If you want to dig in and try to figure out what this example is testing and fix it up, feel free to! But for this PR just to update the Flow types, let's keep the behavior the same by doing `(isEnabled: boolean)` here and not changing the setState line below.

Reply:
Oh yeah, my bad! that was a newbie mistake 🤦‍♂️ . Let's go with adding flow types for now. Will keep the behavior the same and focus on updating flow types.